packages/react/package.json

```diff
@@ -92,6 +93,7 @@
     "tsconfig": "workspace:*",
     "vite": "^3.1.0",
     "vitest": "^0.25.2",
+    "vitest-canvas-mock": "^0.3.3",
```

Comment:
Do we still need this?

Reply:
yes, it's still needed to fix an error in the unit tests coming from [country-flag-emoji-polyfill](https://github.com/talkjs/country-flag-emoji-polyfill) related to `HTMLCanvasElement.prototype.getContext`:
![image](https://github.com/slashid/javascript/assets/29843674/62e680c4-009b-41f2-8314-322acda18ecc)